recipes/popdel/build.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
-make all
+VERSION=1.0.5
+DATE=2018-12-06
```

Comment:
It is displayed as "Date of last update" in the help message of the program. It's not a crucial info but setting it to the build date would be a bit misleading in my opinion. I might just completely remove it from the program in the next version  because the version number provides enough information and if this makes the package easier to maintain that's definitely an argument.

Reply:
Ah, you are upstream author as well. Ok.

I would suggest hard coding those values in the source in this case. It's just a few lines of bash to automate that as part of your release process. There are tools for this too, e.g. `bumpversion`, which will bump your version number, create a commit with message, and create a tag. 

Have a look at my SINA repository (https://github.com/epruesse/SINA). I'm usually bumping from `1.2.3.4-dev` to `1.2.3.4` to `1.2.3.5-dev`, with the tag set on the commit that has `1.2.3.4` as version. That way, wherever people get the source from, I've got an idea later on what version they were using. I'm building the tag on Travis, to make sure all is going ok, and then push the resolved code (with the compiled configure script etc) and osx/linux binaries to Github from there. 

I like the date idea too. It's often helpful to know from which time frame a software is.

Reply:
Plus, that way, we can add a test here `popdel --version | grep ${PKG_VERSION}` to make sure we got the right code. :)

Reply:
That sounds very useful and I will integrate that in the next release. Also the tip with the version check in the test. But because version 1.0.5 is already released I cannot make the necessary changes to the code now.

Reply:
Ok.